app/controllers/profiles_controller.rb

```diff
@@ -1,5 +1,31 @@
 class ProfilesController < ApplicationController
   def my_profile
+    # Gets current user coordinates, currently using static postal code until we fix the Js issue.
+    @current_postal_code = 'H2T 1X3'
```

Comment:
hard coded, we'll just need to make sure it matches all other occurrences in the project so the distance doesn't change.